doc/CONTRIBUTING.md

```diff
@@ -0,0 +1,36 @@
+# Contributing
```

Comment:
Could you include the content from CONTRIBUTING.md in the other SDKs? This talks about the CLA and Community Guidelines.